### PR TITLE
Misc. formatting (Data Tiers/Partitions, kebab menu references)

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request...

- Fixes formatting in Partitions/Data tiers docs to prepare for upcoming Flex Tier release.
- Fixes inconsistent references to three-dot kebab menu.

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/cse/schema/parser-editor.md [hunk 1]
```diff
@@ -156,26 +156,26 @@ Another common reason to set up a local configuration is to pre-parse the conten
 To create a local configuration:
 
 1. Go to **Manage Data** > **Logs** > **Parsers**.
-1. In the System folder, navigate to the parser you want to modify and choose **Edit** from the three-dot more options menu.<br/><img src={useBaseUrl('img/cse/three-dot.png')} alt="Three dot" width="800"/>
+1. In the System folder, navigate to the parser you want to modify and choose **Edit** from the three-dot kebab menu.<br/><img src={useBaseUrl('img/cse/three-dot kebab.png')} alt="three-dot" width="800"/>
```
Comment (kimsauce) [inline docs/cse/schema/parser-editor.md:159]:
```suggestion
1. In the System folder, navigate to the parser you want to modify and choose **Edit** from the three-dot kebab menu.<br/><img src={useBaseUrl('img/cse/three-dot.png')} alt="three-dot kebab" width="800"/>
```

File: docs/dashboards/export-dashboard-new.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ You can generate and download your Dashboard dashboards in PDF, PNG, and JSON f
 To export a Dashboard follow these steps:
 
 1. Open the Dashboard you need to export.
-1. Click the three-dot menu icon on the top right of the Dashboard.<br/><img src={useBaseUrl('/img/dashboards-new/export-dashboard-new/three-dot-icon.png')} alt="three dot icon" style={{border: '1px solid gray'}} width="700" />
+1. Click the three-dot kebab menu icon on the top right of the Dashboard.<br/><img src={useBaseUrl('/img/dashboards-new/export-dashboard-new/three-dot kebab-icon.png')} alt="three-dot icon" style={{border: '1px solid gray'}} width="700" />
```
Comment (kimsauce) [inline docs/dashboards/export-dashboard-new.md:15]:
```suggestion
1. Click the three-dot kebab menu icon on the top right of the Dashboard.<br/><img src={useBaseUrl('/img/dashboards-new/export-dashboard-new/three-dot-icon.png')} alt="three-dot icon" style={{border: '1px solid gray'}} width="700" />
```

File: docs/manage/partitions-data-tiers/index.md [hunk 3]
```diff
@@ -27,7 +28,7 @@ You define the data that will reside in a partition by defining a routing expres
 
 ## Limitations 
 
-* To create and manage partitions, you must be an Admin or you must have the Manage Partitions [role capability](../users-roles/roles/role-capabilities.md). 
+* <PartitionPermissions/>
```
Comment (jpipkin1) [inline docs/manage/partitions-data-tiers/index.md:31]: For some reason, in this file when the reuse content is placed in a bulleted list, it leaves a large space beneath it. Maybe you should leave it outside the bulleted list to see if it behaves better.

**Review by jpipkin1 — APPROVED** [review]
Looks great, @kimsauce! I found only one thing that needs investigating.

File: docs/manage/partitions-data-tiers/index.md (hunk 3 above)
Comment (kimsauce) [reply docs/manage/partitions-data-tiers/index.md:31]: good call! I missed this. Just fixed (got rid of the /reuse file for a separate reason).

**Pull request merged**